apph-front/src/tests/components/DisplayPhoto.test.tsx

```diff
@@ -25,8 +25,11 @@ describe("Display Folder's Photo Tests", () => {
       cookies.set('user', { ...decodedToken, token: JWS_TOKEN });
     }
     //WHEN
-    render(<DisplayPhoto selectedFolder="1" />);
-    expect(screen.getByText(/photo/)).toBeInTheDocument();
+    render(<DisplayPhoto rootFolder="0" selectedFolder="1" />);
+    setTimeout(
+      () => expect(screen.getByText(/photo/)).toBeInTheDocument(),
+      500
+    );
```

Comment:
```suggestion
    expect(await screen.findByText(/photo/)).toBeInTheDocument();
```

utilise findBy plutôt quand tu veux tester un changement qui peut prendre du temps à arriver
![image](https://user-images.githubusercontent.com/39133219/173577402-8680d052-2378-4d69-9729-59a683608849.png)